pages/curriculum/[lessonSlug]/[subLessonSlug].tsx

```diff
@@ -26,25 +26,45 @@ import ScrollTopArrow from '../../../components/ScrollTopArrow'
 import Title from '../../../components/Title'
 import matter from 'gray-matter'
 import useBreakpoint from '../../../helpers/useBreakpoint'
+import { Accordion, Card, useAccordionButton } from 'react-bootstrap'
 
-interface Props {
-  selectedSubLessonIndex: number
-  lessonSlug: string
-  subLessonSlug: string
-  lesson: Omit<GetLessonsQuery['lessons'], 'challenges'>
-  subLessons: SubLesson[]
-  githubFilePath: string
+interface CustomToggleProps {
+  eventKey: string
+  breakpoint: boolean
+  setToggle: React.Dispatch<React.SetStateAction<boolean>>
 }
-const SubLessonPage: React.FC<Props> & WithLayout = ({
-  selectedSubLessonIndex,
-  lessonSlug,
-  subLessonSlug,
-  subLessons,
-  githubFilePath
+const CustomToggle: React.FC<PropsWithChildren<CustomToggleProps>> = ({
```

Comment:
The only reason I created a custom toggle is to control when the accordion is closed/opened and prevent it from getting opened on desktop view.